css/style.css

```diff
@@ -16,4 +27,128 @@ body {
  * - When using Flexbox, remember the items you want to move around need to be inside a parent container set to 'display: flex'
  */
 
+/* styling logo */
+
+/* #karma-logo {
+  margin: 0 1rem 0 1rem;
+} */
```

Comment:
Cosmetics: It's good to remove all the unnecessary comments removed from your code before making a pull request.